packages/astro/package.json

```diff
@@ -3,7 +3,6 @@
 	"type": "module",
 	"version": "0.0.1",
 	"scripts": {
-		"prepare": "panda codegen",
```

Comment:
Bun executes `prepare` after `install`, which is a problem when installing only production dependencies if the `prepare` script depends on dev dependencies.

Reply:
good catch!